File: .sdk/tm/php/utility/MakeContext.php
<?php
declare(strict_types=1);

// Hook0 SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class Hook0MakeContext
{
    public static function call(array $ctxmap, ?Hook0Context $basectx): Hook0Context
    {
        return new Hook0Context($ctxmap, $basectx);
    }
}

File: .sdk/tm/csharp/utility/MakeContext.cs
// Hook0 SDK utility: makeContext.

namespace Hook0Sdk.Util;

public static partial class SdkUtility
{
    internal static Context MakeContextUtil(Dictionary<string, object?>? ctxmap, Context? basectx)
    {
        return new Context(ctxmap, basectx);
    }
}
